i3-agenda.py

```diff
@@ -94,6 +94,9 @@ def get_event_time(full_time):
     else: 
         format = '%Y-%m-%d'
 
+    if full_time[-3] == ":":
```

Comment:
that's a shame :sweat_smile:  
can you add a comment explaining this briefly? otherwise it just kinda looks out of place :)

Reply:
Sure, i've add a small comment in the code